force-app/main/default/classes/RD2_QueueableService.cls

```diff
@@ -94,8 +94,9 @@ public class RD2_QueueableService {
         }
 
         EvaluateInstallmentOpportunities service = new EvaluateInstallmentOpportunities(toProcess, rdIdsWhereScheduleChanged, oldRds);
+        RD2_Settings settings = RD2_Settings.getInstance();
 
-        if (Limits.getQueueableJobs() < Limits.getLimitQueueableJobs()) {
+        if (Limits.getQueueableJobs() < Limits.getLimitQueueableJobs() && !settings.isGiftEntryMode) {
```

Comment:
Is it worth renaming `isGiftEntryMode` to have something to do with this async logic? It looks like it's only being used to determine if we can enqueue a new job... 
Also, there's an unrelated reference to `isGiftEntryMode` in the `GE_GiftEntry` component, confused me for a second 😅